resolwe/test/tests.py

```diff
@@ -1,73 +1,86 @@
+        setattr(self.dummy_case, 'assertEqual', self.assertEqual)
+
     @mock.patch("os.path.isfile")
-    @mock.patch("resolwe.test.testcases.process.dict_dot")
-    def test_assert_file_date_nofilter(self, dict_dot_mock, isfile_mock):
+    def test_assert_file_date_nofilter(self, isfile_mock):
         isfile_mock.return_value = True
         output1_file = io.BytesIO(b"some line\ndate: 2016-02-10\n")
         output2_file = io.BytesIO(b"some line\ndate: 2015-10-31\n")
         open_mock = mock.MagicMock(side_effect=[output1_file, output2_file])
         with mock.patch.object(builtins, 'open', open_mock):
-            dummy_case = ProcessTestCase.__new__(ProcessTestCase)
-            dummy_case.files_path = ''
-            dummy_case._debug_info = lambda _: ''  # pylint: disable=protected-access
-            setattr(dummy_case, 'assertEqual', self.assertEqual)
             obj_mock = mock.MagicMock()
             self.assertRaises(AssertionError, ProcessTestCase._assert_file,  # pylint: disable=protected-access
-                              dummy_case, obj_mock, "", "")
+                              self.dummy_case, obj_mock, "", "")
 
     @mock.patch("os.path.isfile")
-    @mock.patch("resolwe.test.testcases.process.dict_dot")
-    def test_assert_file_date_filter(self, dict_dot_mock, isfile_mock):
+    def test_assert_file_date_filter(self, isfile_mock):
         isfile_mock.return_value = True
         output1_file = io.BytesIO(b"some line\ndate: 2016-02-10\n")
         output2_file = io.BytesIO(b"some line\ndate: 2015-10-31\n")
         open_mock = mock.MagicMock(side_effect=[output1_file, output2_file])
         with mock.patch.object(builtins, 'open', open_mock):
-            dummy_case = ProcessTestCase.__new__(ProcessTestCase)
-            dummy_case.files_path = ""
-            dummy_case._debug_info = lambda _: ""  # pylint: disable=protected-access
-            setattr(dummy_case, 'assertEqual', self.assertEqual)
             obj_mock = mock.MagicMock()
 
             def date_in_line(line):
                 return line.startswith(b"date")
 
-            ProcessTestCase._assert_file(dummy_case, obj_mock, "", "",  # pylint: disable=protected-access
+            ProcessTestCase._assert_file(self.dummy_case, obj_mock, "", "",  # pylint: disable=protected-access
                                          file_filter=date_in_line)
 
-    @unittest.skipIf(six.PY2, "Needs Python 3 version of the gzip module")
+    @mock.patch("os.path.isfile")
+    def test_assert_file_sort_none(self, isfile_mock):
+        isfile_mock.return_value = True
+        output1_file = io.BytesIO(b"A\nB\nC\n")
+        output2_file = io.BytesIO(b"A\nC\nB\n")
+        open_mock = mock.MagicMock(side_effect=[output1_file, output2_file])
+        with mock.patch.object(builtins, 'open', open_mock):
+            obj_mock = mock.MagicMock()
+            self.assertRaises(AssertionError, ProcessTestCase._assert_file,  # pylint: disable=protected-access
```

Comment:
ProcessTestCase._assert_file(  # pylint: disable=protected-access
        self.dummy_case, obj_mock, "", "", sort=lambda x: str(x).lower()
    )

Also change this in tests below.